extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -92,29 +92,33 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     public void initialize() {
 
         try {
-            logger.debug("Initializing NTP handler for '{}'.", getThing().getUID());
+            logger.debug("Initializing NTP handler for '{}'.", getThing().getUID().toString());
 
             Configuration config = getThing().getConfiguration();
-            this.hostname = (String) config.get(PROPERTY_NTP_SERVER);
-            this.refreshInterval = (BigDecimal) config.get(PROPERTY_REFRESH_INTERVAL);
-            this.refreshNtp = (BigDecimal) config.get(PROPERTY_REFRESH_NTP);
-            this.refreshNtpCount = 0;
+            hostname = (String) config.get(PROPERTY_NTP_SERVER);
+            refreshInterval = (BigDecimal) config.get(PROPERTY_REFRESH_INTERVAL);
+            refreshNtp = (BigDecimal) config.get(PROPERTY_REFRESH_NTP);
+            refreshNtpCount = 0;
 
             try {
                 timeZone = TimeZone.getTimeZone((String) config.get(PROPERTY_TIMEZONE));
             } catch (Exception e) {
                 timeZone = TimeZone.getDefault();
-                logger.debug("using default TZ: {}", timeZone);
+                logger.debug("{} using default TZ: {}", getThing().getUID().toString(), timeZone);
             }
 
             try {
                 String localeString = (String) config.get(PROPERTY_LOCALE);
                 locale = new Locale(localeString);
             } catch (Exception e) {
                 locale = Locale.getDefault();
-                logger.debug("using default locale: {}", locale);
+                logger.debug("{} using default locale: {}", getThing().getUID().toString(), locale);
             }
             dateTimeChannelUID = new ChannelUID(getThing().getUID(), CHANNEL_DATE_TIME);
+            updateStatus(ThingStatus.OFFLINE);

```

Comment:
do you need to go OFFLINE here or is it enough that the automatic refresh determines whether it is ONLINE or OFFLINE afterwards?